packages/suite/src/views/wallet/coinmarket/DCA/index.tsx

```diff
@@ -0,0 +1,164 @@
+import React from 'react';
+import {
+    Column,
+    Divider,
+    H2,
+    Icon,
+    IconType,
+    Image,
+    Paragraph,
+    Row,
+    Text,
+} from '@trezor/components';
+import { UseCoinmarketProps } from 'src/types/coinmarket/coinmarket';
+import { withSelectedAccountLoaded } from 'src/components/wallet';
+import { CoinmarketLayout } from '../common';
+import styled from 'styled-components';
+import { borders, colorVariants, palette, spacings, spacingsPx } from '@trezor/theme';
+import { Translation, TrezorLink } from 'src/components/suite';
+import { COINMARKET_DOWNLOAD_INVITY_APP_URL } from '@trezor/urls';
+import { useSelector } from 'src/hooks/suite';
+import { variables } from '@trezor/components/src/config';
+
+const IconWrapper = styled.div`
+    display: flex;
+    flex-shrink: 0;
+    justify-content: center;
+    align-items: center;
+    width: 40px;
+    height: 40px;
+    border-radius: 50%;
+    background: ${palette.lightSecondaryLime500};
+    margin-right: ${spacingsPx.sm};
+`;
+
+const WrappedText = styled.div`
+    max-width: 200px;
+    text-align: center;
+`;
+
+const ColumnsWrapper = styled.div`
+    ${variables.SCREEN_QUERY.ABOVE_MOBILE} {
+        display: flex;
+    }
+`;
+
+const Column1 = styled.div`
+    display: flex;
+    flex-direction: column;
+    flex: 1 0 50%;
+    gap: ${spacingsPx.xxl};
+    background-color: ${colorVariants.standard.backgroundPrimaryDefault};
+    padding: ${spacingsPx.xxl};
+    border-radius: ${borders.radii.md};
+    color: ${colorVariants.standard.textOnPrimary};
+`;
+
+const Column2 = styled.div`
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    flex: 1 0 50%;
+    gap: ${spacingsPx.md};
+    padding: ${spacingsPx.xxl};
+`;
+
+const Layout = styled(CoinmarketLayout)`
+    & [class^='CoinmarketLayout__Content'] {
```

Comment:
Could you please refactor this solution? It's very hard to read and I'm not sure if it's working on production build because classnames are obfuscated there (it's not happening in dev build).

Reply:
changed